src/main/java/org/jitsi/stats/media/AbstractStatsPeriodicRunnable.java

```diff
@@ -107,40 +107,29 @@ public AbstractStatsPeriodicRunnable(
             }
         }
 
-        try
-        {
-            EntityBareJid roomJid = JidCreate.entityBareFrom(conferenceJid);
-
-            String conferenceName = roomJid.getLocalpart().toString();
+        String conferenceName = conferenceJid.getLocalpart().toString();
 
-            // extract siteId/subdomain
-            String domain = roomJid.getDomain().toString();
-            if (conferenceIDPrefix != null
-                && domain.endsWith("." + conferenceIDPrefix))
+        // extract siteId/subdomain
+        String domain = conferenceJid.getDomain().toString();
+        if (conferenceIDPrefix != null
+            && domain.endsWith("." + conferenceIDPrefix))
+        {
+            // strip `.conferenceIDPrefix`
+            String mucDomain = domain.substring(0, domain.indexOf(conferenceIDPrefix) - 1);
+            int dotIx = mucDomain.indexOf('.');
+            if (dotIx > 0)
             {
-                // strip `.conferenceIDPrefix`
-                String mucDomain = domain.substring(0, domain.indexOf(conferenceIDPrefix) - 1);
-                int dotIx = mucDomain.indexOf('.');
-                if (dotIx > 0)
-                {
-                    String siteId = mucDomain.substring(dotIx + 1);
+                String siteId = mucDomain.substring(dotIx + 1);
 
-                    if (!StringUtils.isNullOrEmpty(siteId))
-                    {
-                        this.initiatorSiteID = siteId;
-                        conferenceIDBuilder.append(siteId).append("/");
-                    }
+                if (!StringUtils.isNullOrEmpty(siteId))
```

Comment:
Can you use `isNotBlank` from apache-commons please? We've been trying to phase-out our StringUtils